applications/GeoMechanicsApplication/tests/line_interface_elements/Neumann_multi_stage_reset_displacements/ProjectParameters_stage1.json

```diff
@@ -0,0 +1,152 @@
+                        "file_label":          "step",
+                        "output_control_type": "step",
+                        "output_interval":    1,
+                        "body_output":         true,
+                        "node_output":         true,
+                        "skin_output":         false,
+                        "plane_output":        [],
+                        "nodal_results":       ["DISPLACEMENT"],
+                        "gauss_point_results": []
+                    },
+                    "point_data_configuration":  []
+                }
+            }
+        }]
+    },
+    "processes": {
+        "constraints_process_list": [{
+            "python_module": "apply_vector_constraint_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name":  "ApplyVectorConstraintTableProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.FixedNodes",
+                "variable_name":   "DISPLACEMENT",
+                "active":          [true,true,false],
+                "is_fixed":        [true,true,true],
+                "value":           [0.0,0.0,0.0],
+                "table":           [0,0,0]
+            }
+        },{
+            "python_module": "apply_scalar_constraint_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name":  "ApplyScalarConstraintTableProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.AllNodes",
+                "variable_name":   "WATER_PRESSURE",
+                "active":          true,
+                "is_fixed":        true,
+                "value":           0.0,
+                "table":           0
+            }
+        }],
+        "loads_process_list": [{
+            "python_module": "apply_normal_load_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name":  "ApplyNormalLoadTableProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.LoadedSide",
+                "variable_name":   "NORMAL_CONTACT_STRESS",
+                "active":          [true,false],
+                "value":           [0.0,0.0],
+                "table":           [2,0],
+                "fluid_pressure_type": "Uniform"
+            }
+        },{
+            "python_module": "apply_normal_load_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name":  "ApplyNormalLoadTableProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.LoadedSide",
+                "variable_name":   "TANGENTIAL_CONTACT_STRESS",
+                "active":          [false,true],
+                "value":           [0.0,667.0],
+                "table":           [0,1],
+                "fluid_pressure_type": "Uniform"
```

Comment:
Preferly to have "fluid_pressure_type" before "value". It makes it easier to follow. However, it is matter of preference.

Reply:
I agree, moved!